prow.sh

```diff
@@ -846,102 +855,99 @@ main () {
         run_with_go "${CSI_PROW_GO_VERSION_BUILD}" make container || die "'make container' failed"
     fi
 
-    install_kind || die "installing kind failed"
-    start_cluster || die "starting the cluster failed"
+    if test_enabled "sanity" || test_enabled "parallel" || test_enabled "serial" || test_enabled "serial-alpha" || test_enabled "parallel-alpha"; then
```

Comment:
How about having a couple of variables that can serve as whitelists of jobs that:

* require non-alpha cluster
* require alpha cluster

Reply:
You mean functions like `job_needs_cluster`, `job_needs_alpha_cluster`, `job_needs_non_alpha_cluster` instead of spelling that out via `test_enabled`? Yes, that would be a bit cleaner, albeit not necessarily shorter overall. I'll change it.